chipsec/hal/spd.py

```diff
@@ -364,7 +364,7 @@ def detect( self ):
         _dimms = []
         for d in SPD_DIMMS:
             if self.isSPDPresent( d ): _dimms.append( d )
-        if logger().HAL: 
+        if logger().HAL:
             logger().log( "Detected the following SPD devices:" )
             for _dimm in _dimms: logger().log( "%s: 0x%02X" % (SPD_DIMMS[_dimm],_dimm) )
```

Comment:
If you're modifying this file, might as well also update this from % to format?

Reply:
Just a white space change so no need to do anything else.